runtime/coverage.go

```diff
@@ -120,6 +120,9 @@ type CoverageReport struct {
 	// This filter can be used to inject custom logic on
 	// each location/program inspection.
 	LocationFilter LocationFilter `json:"-"`
+	// Contains a mapping with human-friendly names for
+	// locations. This could be the filepath of a location.
```

Comment:
I guess this mapping holds the "source path" rather than a "human-friendly name", against the location?
Maybe better to put it that way in the description?

Reply:
Updated in https://github.com/onflow/cadence/pull/2859/commits/4871d50d108d715a582746143ff09d72e78c525b